spec/factories/okta_profiles.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :okta_idme_response, class: Hash do
+    profile do
+      {
+        'lastName' => 'CARROLL',
+        'gender' => 'F',
+        'secondEmail' => null,
```

Comment:
should this be `nil`?

Reply:
someone's been shifting a lot between Node and Ruby lol